### ISSUE TITLE
Firefox Exclusive Issue - Placing Focus on a Combobox Hides its Content

### ISSUE THREAD

**Issue by Santoz2** [ManifoldScholar/manifold#2557]
Note, this issue is specifically for Firefox. I could not reproduce this on any other browser. 

## Expected Behavior
The text of interactable elements should still be visible when they are focused. In this case, the text of a Combobox should be left visible when a Combobox is focused.

## Current Behavior
When using Firefox, in Manifold's Admin Interface, focusing a Combobox with a keyboard visually hides the currently selected text for that Combobox. This issue was not present in Microsoft Edge, Google Chrome, or Safari.

![image](https://user-images.githubusercontent.com/43553368/72461967-e8d97e80-3784-11ea-814c-93760b3d294c.png)

## Steps to Reproduce
1. Navigate to the Admin Interface (via Enter Admin Mode)
2. Navigate to an editable project, landing on its General Page
3. Navigate to the Publication Date field
4. Notice how the text disappears on focus

## Testing Environment
NVDA 2019.2.1
ESR Firefox 68.4.1 esr

**Comment by dananjohnson** [comment]
Thanks for flagging this, @Santoz2. It looks like the color is being inherited from the non-standard `:-moz-focusring` pseudo-class, which is set to `transparent`. Should be an easy fix I think.

**Comment by Santoz2** [comment]
Verified as fixed, great job!
